modules/core/src/main/scala/lucuma/odb/api/model/SpectroscopyRequirements.scala

```diff
@@ -0,0 +1,210 @@
+// Copyright (c) 2016-2021 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.odb.api.model
+
+import cats.Eq
+import cats.syntax.all._
+import cats.data.State
+import clue.data.Input
+import eu.timepit.refined._
+import eu.timepit.refined.numeric.Positive
+import eu.timepit.refined.cats._
+import eu.timepit.refined.types.numeric.PosBigDecimal
+import eu.timepit.refined.types.numeric.PosInt
+import io.circe.Decoder
+import io.circe.generic.semiauto._
+import io.circe.refined._
+import lucuma.odb.api.model.FocalPlane
+import lucuma.odb.api.model.SpectroscopyCapabilities
+import lucuma.odb.api.model.syntax.input._
+import lucuma.core.math.Angle
+import lucuma.core.util.Display
+import lucuma.core.math.Wavelength
+import lucuma.core.util.Enumerated
+import lucuma.core.optics.syntax.lens._
+import monocle.Focus
+import monocle.Lens
+
+final case class SpectroscopyScienceRequirements(
+  wavelength:       Option[Wavelength],
+  resolution:       Option[PosInt],
+  signalToNoise:    Option[PosBigDecimal],
+  signalToNoiseAt:  Option[Wavelength],
+  wavelengthRange:  Option[Wavelength],
+  focalPlane:       Option[FocalPlane],
+  focalPlaneAngle:  Option[Angle],
+  capabilities:     Option[SpectroscopyCapabilities]
```

Comment:
Ignorance revealing question: can you do multiple capabilities in the same observation?

Reply:
Not as specified now, We could of course make this a list for the eventuality?

Reply:
Nah, we can change it later if need be.